iquip/apps/builder.py

```diff
@@ -456,20 +460,33 @@ def initArgsEntry(self, experimentInfo: ExperimentInfo):
         Args:
             experimentInfo: The experiment information.
         """
+        nonScanEntry = ["BooleanValue", "StringValue", "EnumerationValue", "NumberValue"]
         for argName, (argInfo, *_) in experimentInfo.arginfo.items():
             # TODO(BECATRUE): The other types such as 'Scannable'
```

Comment:
Now, this line looks unnecessary.

Reply:
I missed it. I will add it at next PR.

Reply:
I used it for case diverge at line 467.
467: `if argType in nonScanEntry:`

Reply:
How about moving original entryCls at line 468 to the position of nonScanEntry?

Reply:
Could you remove these two comments?